digital-twins/jsonld-contexts/digital_twins_compound.jsonld

```diff
@@ -1,5 +1,6 @@
 {
   "@context": [
+    "https://raw.githubusercontent.com/easy-global-market/c2jn-data-models/main/jsonld-contexts/c2jn-compound.jsonld",
```

Comment:
`c2jn-compound` is already including dcat, authorization and core context. not so clean to include them twice (and can possibly cause side effects).

maybe one way to do it:
- create a c2jn-smart-city-compound context in c2jn-data-models project with only smart city specific contexts
- (later) create a c2jn-agile-company-compound context in c2jn-data-models project with only agile company specific contexts
- include here c2jn-smart-city-compound context (and later the other one)
- update the c2jn-compound context in c2jn-data-models project to include the two aforementioned specific business contexts for other users in the project